### PR TITLE
V2.8.2 release/jitpack support

### PR DESCRIPTION
**Pull request by BhavaniPrasadUserIQ**
UserIQ SDK Version 2.8.2 release - Added Jitpack Support

### CODE REVIEW

File: Sample app/app/build.gradle [hunk 1]
```diff
@@ -22,8 +22,8 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:28.0.0-rc02'
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation 'com.android.support:design:28.0.0-rc02'
-    implementation 'com.useriq:sdk:2.6.0'
-
+    // implementation 'com.useriq:sdk:2.6.0'
```
Comment (sudhakar) [inline Sample app/app/build.gradle:25]:
```suggestion
```
Why is old comment left behind?

File: jitpack.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+install:
+  - FILE="-Dfile=SDK/sdk-2.8.2.aar"
+  - mvn install:install-file $FILE -DgroupId=ccom.github.useriq-com -DartifactId=android-sdk -Dversion=2.8.2 -Dpackaging=aar -DgeneratePom=true
```
Comment (sudhakar) [inline jitpack.yml:3]:
```suggestion
  - mvn install:install-file $FILE -DgroupId=com.github.useriq-com -DartifactId=android-sdk -Dversion=2.8.2 -Dpackaging=aar -DgeneratePom=true
```

**Review by sudhakar — CHANGES_REQUESTED** [review]
Make changes

File: Sample app/app/build.gradle (hunk 1 above)
Comment (BhavaniPrasadUserIQ) [reply Sample app/app/build.gradle]: Removed old comment.

File: jitpack.yml (hunk 2 above)
Comment (BhavaniPrasadUserIQ) [reply jitpack.yml]: Updated DgroupID, Thanks!

**Review by sudhakar — APPROVED** [review]

**Pull request merged**